digit-health-services/libraries/health-services-common/src/main/java/org/digit/health/common/data/mapper/GenericRowMapper.java

```diff
@@ -0,0 +1,95 @@
+package org.digit.health.common.data.mapper;
+
+import org.digit.health.common.utils.ObjectUtils;
+import org.springframework.beans.BeanUtils;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.JdbcUtils;
+
+import java.lang.reflect.Field;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * GenericRowMapper is an implementation of RowMapper<T>. It is used for mapping ResultSet from sql query to List<T><p>
+ * GenericRowMapper throws SQLException if there is an error related to ResultSet, db or query.<p>
+ * GenericRowMapper throws RuntimeException whenever there is an error while converting resultset to object.<p>
+ *<p>
+ *<p>
+ * Requirements to use GenericRowMapper<p>
+ * * Column name and member variable name must be similar. for example columnName "price" will be mapped to member variable named "price"<p>
```

Comment:
Make F capital for "for example" and put a comma after it.
"For example, "

Reply:
Updated